bin/offline-dbg

```diff
@@ -133,6 +142,129 @@ EOF
     echo "**************************************************************"
 }
 
+check_for_OpenFlow_14() {
+    # If no protocol listed, default to atomic bundle txn (enabled by default on ovs 2.8 and on)
+    local hex_ofp_version=$(cat ${CMD_DIR}/ovs-ofctl_-t_5_--version | grep "OpenFlow versions" | grep -oP [0-9]*x[0-9]* | tail -1)
+    local version=${open_flow_protocols[$hex_ofp_version]}
+    if [ -z "$version" ]; then
+        echo " --bundle"
+    else
+        [ ${version#OpenFlow} -ge 14 ] && bundle=" --bundle" || bundle=""
+        echo $bundle
+    fi                    
+}
+
+do_collect-sos() {
+    if [ $# -lt 1 ]; then
+        usage
+        exit 1
+    fi
+    local sos=$1
+
+    local dir_name=`tar -tf $sos | head -1 | cut -f1 -d"/"`
+
+    local sos_files="${dir_name}/sos_commands/rpm ${dir_name}/sos_commands/openvswitch"
+
+    local db_locations="var/lib/openvswitch etc/openvswitch usr/local/etc/openvswitch"
+
+    if test -f "$sos"; then
+        mkdir ${WORKDIR}/sos_report
```

Comment:
Are you willing to support multiple sos_report extracted in the same workdir?
if the answer is no, then properly fail if $workdir/sos_report already exists and ask the user to run "stop" before or change workdir. Currently it just outputs:

mkdir: cannot create directory ‘/tmp/ovs-offline-dbg/sos_report’: File exists